app/controllers/manageiq/api/base_controller.rb

```diff
@@ -105,7 +101,7 @@ def set_access_control_headers
       extend ErrorHandler::ClassMethods
       respond_to :json
       rescue_from_api_errors
-      prepend_before_action :require_api_user_or_token, :except => [:handle_options_request]
+      prepend_before_action :require_api_user_or_token
```

Comment:
Ok to secure OPTIONS for /api/:collection, but I'm not sure we want to secure OPTIONS get on /api itself  (CORS stuff), /cc @martinpovolny ?